### DISCUSSION TITLE
doubts about licensing

### DISCUSSION THREAD

**Discussion by pdesterlich** [SixLabors/Fonts#353] [General] [answered]
it may be stupid, but i was thinking about your licensing model which states that "you're subject to Apache License v2 if you consume the library as a Transitive Package Dependency"

from what i understand, that means that if i use package x, let say ClosedXML, and package x has SixLabor Fonts as a dependency, i'm under Apache License no matter the size of my business if i use it for a commercial software.

if this is true, doesn't this mean that i can create an open source package that does little to nothing but have SixLabor Fonts as a dependency, publish it on nuget and then use it in my application. This way i would have SixLabor fonts as a transitive dependency but still usable, and technically according to your licence i should not have to pay for a commercial licence.

Am i wrong or can this be a possible flaw in the license model?

**Comment by JimBobSquarePants** [comment, marked as answer]
It's by design. See the relevant comment in the full discussion here.

https://github.com/SixLabors/ImageSharp/discussions/2151#discussioncomment-2942963
